library/unistd/common_pathconf.c

```diff
@@ -117,12 +117,12 @@ __pathconf(struct MsgPort *port, int name) {
             ret = 510;    /* I could not find any documentation regarding this. */
             break;
         case _PC_MAX_INPUT: {
-                uint32_t Bufsize;
+                uint32_t Bufsize = 0L;
```

Comment:
The `Bufsize` was not initialised and later, based on conditions, an initial value is required, but I am not sure about the value I put here. Is that right?

Reply:
Should be UL, but it depends on the architecture if L is actually needed. int is normally a targets natural word size, but also can be as low as 16bits. Perhaps U would work?

Reply:
@3246251196 With my last commit, I changed this to 2048, because a few lines later (line 127) there is this comment `ret = Bufsize;    /* Default is 2048 bytes. */`